CredentialProvider/CredentialAssets.xcassets/Colors/credentialCellColor.colorset/Contents.json

```diff
@@ -0,0 +1,56 @@
+{
+  "colors" : [
+    {
+      "color" : {
+        "color-space" : "srgb",
+        "components" : {
+          "alpha" : "1.000",
+          "blue" : "1.000",
+          "green" : "1.000",
+          "red" : "1.000"
+        }
+      },
+      "idiom" : "universal"
+    },
+    {
+      "appearances" : [
+        {
+          "appearance" : "luminosity",
+          "value" : "light"
```

Comment:
Are we using colors in xcassets so we can use them in the `.storyboard`?

Reply:
We may not be able to use appearances until we drop support for iOS 11/12.

Reply:
@boek Isn't that backward compatible? Before iOS 13 there is no dark/light mode so the UI defaults to light and these assets are not touched/used?